test/WebJobs.Script.Tests/Workers/Rpc/RpcWorkerProcessTests.cs

```diff
@@ -62,6 +63,23 @@ public RpcWorkerProcessTests()
                 testEnv,
                 scriptApplicationHostOptions,
                 new LoggerFactory());
+
+            _rpcWorkerProcessWithExecutablePath = new RpcWorkerProcess("node",
```

Comment:
Any reason why this needs to be a field? Seems like you can just define this locally in your CreateWorkerProcess_AddsExecutableWorkingDirectory test below, since it's not used anywhere else

Reply:
I allows re-using other mocks defined in scope

Reply:
Unless you're reusing it, just define it locally in the test. If/when we ever need to reuse in other tests, it can be hoisted at that time.